.github/workflows/lint-test.yml

```diff
@@ -12,6 +12,9 @@ concurrency:
 
 jobs:
   lint-test:
+    continue-on-error: true
```

Comment:
I don't know we should necessarily do this. A lot of our actions have a very linear flow of steps that depend on each other:
Install dependencies -> setup requirements and environment -> do something

In theory it doesn't matter if one step fails and causes the next to fail instead of be skipped, but it doesn't seem appropriate for some of the more critical actions such as deployments and uploading images.

I believe you can force a certain step to always run without forcing the entire workflow to continue though, [example](https://github.com/python-discord/bot/blob/main/.github/workflows/lint-test.yml#L84).

Reply:
The problem is that we don't know which step would fail, and since the second job needs the first one, it should always run otherwise the second might not run? (Not sure of this tbh, I'll test it)

The `continue-on-error` cannot be part of the job that calls the reusable workflow as well, that's why I put it there

Reply:
I just tested it, the `call_status_embed` doesn't trigger without having `continue-on-error` set to true.
It also won't when i move that key to the reusable workflow's file

Reply:
Could you clarify, I don't really follow. Where would the on-error go, and what is the second step. Why does it matter knowing which fails, don't we just care that we run this step if something fails?

Reply:
This has been discussed internally, but for people who might bump into this, here's what has been discussed:

We had the `continue-on-error` option set because we always need the first job to run entirely in order for the second job to trigger and that's because the second depends on the first.

However, it would be a waste to run future steps knowing they would fail if the previous ones did.

The solution for this was to add an `if` statement that will always make the second one run.